generators/database-changelog-liquibase/index.js

```diff
@@ -42,6 +48,23 @@ module.exports = class extends BaseGenerator {
 
     // Set number of rows to be generated
     this.numberOfRows = 10;
+    useBlueprints = !this.fromBlueprint && this.instantiateBlueprints('database-changelog-liquibase');
+  }
+
+  _loading() {
+    return {
+      loadSharedConfig() {
+        this.loadAppConfig();
+        this.loadClientConfig();
+        this.loadServerConfig();
+        this.loadTranslationConfig();
+      },
+    };
+  }
+
+  get loading() {
+    if (useBlueprints) return;
```

Comment:
Blueprint support is not complete need to add `if (useBlueprints) return;` to others priorities.

Reply:
Thanks, forgot about this one: I'll need a crash course on the impact between extending basegenerator and and blueprintbasegen: I do not really understand the impacts (I only know that there are some more steps in the lifecycle)